### DISCUSSION TITLE
Universal member theme setting with a per-member account override.

### DISCUSSION THREAD

**Discussion by PDXMan1** [ExpressionEngine/ExpressionEngine#3070] [Feature Requests]
When I set the CP theme to "slate" using the Jump field, my theme preference is only remembered as long as I use the same device. Each time I log into the CP on a new device, I have to reset the theme all over again.

Once a member sets their theme preference, it should be stored specifically for that member, so that whatever device that member uses displays the CP with that member's theme preference. 

It would be ideal if a Super Admin could set a specific theme as the default for all members using the CP. We do a lot of software training and education using screenshots of the CP. It's a lot easier for our clients to retain information when our screenshots use the same theme they see when they are logged into the CP.

Thank you for your consideration.

**Comment by intoeetive** [comment]
We have an in-progress PR for this: https://github.com/ExpressionEngine/ExpressionEngine/pull/2198
